packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -232,6 +295,66 @@ export abstract class BaseService extends Resource
     return ret;
   }
 
+  public loadBalancerTarget(options: LoadBalancerTargetOptions): IEcsLoadBalancerTarget {
```

Comment:
Yeah, I think this docstring needs a rewrite. Some things I want to highlight:

> This method is called to register the 

It doesn't have a side effect, so I wouldn't phrase it like this.

> Don't call this function alone. 

Avoid using negatives. Tell me what I should be doing instead! 

Suggested phrasing:

```
Return a load balancing target for a specific container and port

Use this function to create a load balancer target if you want to load balance to
another container than the first essential container or the first mapped port on
the container.

Use the return value of this function where you would normally use a load balancer
target, instead of the `Service` object itself.

@example

listener.addTarget(service.loadBalancerTarget({
  containerName: 'MyContainer',
  containerPort: 1234
}));
```

Reply:
Very helpful feedback. Thanks!